README.md

```diff
@@ -315,6 +315,9 @@ The following instructions are intended to help with modifying the codebase.
 Run the `build_and_test.sh` script from the root of the repository to build and
 run the unit tests using the locally installed version of Python.
 
+Run `bazel test tests/cpp:all` from the root of the repository to run unit
```

Comment:
Do you also want to work this into the `build_and_test.sh` script?

Reply:
I thought of that, but the need to install bazel to run the tests would change the requirements to run `build_and_test.sh`.  I'm going to leave them separate for now but may look into integrating the cpp tests into the `build-dist.sh` script.